src/Storefront/Resources/app/storefront/src/plugin/listing/listing-pagination.plugin.js

```diff
@@ -33,9 +33,17 @@ export default class ListingPaginationPlugin extends FilterBasePlugin {
     onChangePage(event) {
         this.tempValue = event.target.value;
         this.listing.changeListing();
+        this._scrollToTop();
```

Comment:
Could you try:
```suggestion
        this.listing.el.scrollIntoView();
```

This could be a solution to the suggestion of @JoshuaBehrens and you woul not need the `_scrollToTop` method.